chrome/content/zotero/locateMenu.js

```diff
@@ -475,7 +475,7 @@ var Zotero_LocateMenu = new function() {
 			if(doi && typeof doi === "string") {
 				doi = Zotero.Utilities.cleanDOI(doi);
 				if(doi) {
-					return "http://dx.doi.org/" + encodeURIComponent(doi);
+					return "https://dx.doi.org/" + encodeURIComponent(doi);
```

Comment:
Should just be `https://doi.org/` these days

(The HTTPS actually makes no difference, because dx.doi.org is in the HSTS preload list.